aws-xray-recorder-sdk-core/src/main/java/com/amazonaws/xray/AWSXRayRecorder.java

```diff
@@ -138,6 +138,10 @@
     private boolean forcedTraceIdGeneration;
 
     public AWSXRayRecorder() {
+        this(null);
+    }
+
+    public AWSXRayRecorder(Emitter sharedEmitter) {
```

Comment:
Not too excited about increasing our API surface for this, maybe we can make the new constructor package private?

Reply:
Sure.